zrml/court/src/migrations.rs

```diff
@@ -0,0 +1,103 @@
+use crate::{Config, Jurors, Pallet};
+use frame_support::{
+    dispatch::Weight,
+    log,
+    pallet_prelude::PhantomData,
+    traits::{Get, OnRuntimeUpgrade, StorageVersion},
+};
+
+const COURT_REQUIRED_STORAGE_VERSION: u16 = 0;
+const COURT_NEXT_STORAGE_VERSION: u16 = 1;
+
+pub struct JurorsConuntedStorageMapMigration<T>(PhantomData<T>);
+
+impl<T: Config> OnRuntimeUpgrade for JurorsConuntedStorageMapMigration<T> {
+    fn on_runtime_upgrade() -> Weight
+    where
+        T: Config,
+    {
+        let mut total_weight = T::DbWeight::get().reads(1);
+
+        if StorageVersion::get::<Pallet<T>>() != COURT_REQUIRED_STORAGE_VERSION {
+            log::info!("Skipping storage migration of jurors of court already up to date");
+            return total_weight;
+        }
+        log::info!("Starting storage migration of jurors of court");
+        Jurors::<T>::initialize_counter();
+        total_weight =
+            total_weight.saturating_add(T::DbWeight::get().writes(Jurors::<T>::count() as u64));
```

Comment:
Plus one for writing the count.

```suggestion
        total_weight = total_weight.saturating_add(                            
            T::DbWeight::get().writes(Jurors::<T>::count().saturating_add(1) as u64),                                                                         
        );
```